cn/index.html

```diff
@@ -445,14 +445,14 @@ <h2 class="subhed">置身代码</h2>
 
       <p class="subhed">这是以下诸位爱与泪的结晶</p>
 
-      <ul class="builders">
-          <li><a href="http://twitter.com/wilto"><img src="http://twitter.com/api/users/profile_image/wilto?size=normal"> <b>Mat</b> Marquis</a>
-          <li><a href="http://twitter.com/aaronforsander"><img src="http://twitter.com/api/users/profile_image/aaronforsander?size=normal"> <b>Aaron</b> Forsander</a>
-          <li><a href="http://twitter.com/connor"><img src="http://twitter.com/api/users/profile_image/connor?size=normal"> <b>Connor Montgomery</b></a>
-          <li><a href="http://twitter.com/paul_irish"><img src="http://twitter.com/api/users/profile_image/paul_irish?size=normal"> <b>Paul</b> Irish</a>
-          <li><a href="http://twitter.com/divya"><img src="http://twitter.com/api/users/profile_image/divya?size=normal"> <b>Divya</b> Manian</a>
-          <li><a href="http://twitter.com/necolas"><img src="http://twitter.com/api/users/profile_image/necolas?size=normal"> <b>Nicolas</b> Gallagher</a>
-          <li><a href="http://twitter.com/addyosmani"><img src="http://twitter.com/api/users/profile_image/addyosmani?size=normal"> <b>Addy</b> Osmani</a>
+       <ul class="builders">
+          <li><a href="https://twitter.com/wilto"><img src="https://pbs.twimg.com/profile_images/477441227986444289/KHCUqTis_normal.png" alt="Wilto"> <b>Mat</b> Marquis</a>
+          <li><a href="https://twitter.com/aaronforsander"><img src="https://pbs.twimg.com/profile_images/461528075788763138/c7ONSIRm_normal.jpeg" alt="Aaronforsander"> <b>Aaron</b> Forsander</a>
+          <li><a href="https://twitter.com/connor"><img src="https://pbs.twimg.com/profile_images/526185574039031808/IpDbRAX5_normal.jpeg" alt="Connor"> <b>Connor Montgomery</b></a>
+          <li><a href="https://twitter.com/paul_irish"><img src="https://pbs.twimg.com/profile_images/420826194083213312/CP1RmLa3_normal.jpeg" alt="Paul Irish"> <b>Paul</b> Irish</a>
+          <li><a href="https://twitter.com/divya"><img src="https://pbs.twimg.com/profile_images/550530231853776897/kni4F8oM_normal.png" alt="Diviya"> <b>Divya</b> Manian</a>
+          <li><a href="https://twitter.com/necolas"><img src="https://pbs.twimg.com/profile_images/529444560179449857/eKdCJhvp_normal.jpeg" alt="Necolas"> <b>Nicolas</b> Gallagher</a>
+          <li><a href="https://twitter.com/addyosmani"><img src="https://pbs.twimg.com/profile_images/422476220442234880/jlx9HMtr_normal.jpeg" alt="addyosmani"> <b>Addy</b> Osmani</a>
```

Comment:
I like the alt attributes, but those should probably just include our full names.